doc/config.rst

```diff
@@ -1088,6 +1088,11 @@ that use Sphinx's HTMLWriter class.
 
    Output file base name for HTML help builder.  Default is ``'pydoc'``.
 
+.. confval:: html_experimental_html5_builder
+
+   Output is processed with HTML5 writer.  This feature needs docutils 0.13.1 or newer.  Default is ``False``.
```

Comment:
The name and descriptions are different. which is correct; `html5_builder` or HTML5 Writer?